accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+# Proposal for global.json updates for preview and roll-forward
+
+---
+**NOTE:** 
+
+SDK version numbers do not use semantic versioning. [Docs has an article on SDK versioning](https://docs.microsoft.com/en-us/dotnet/core/versions/).
+
+---
+
+`global.json` allows users to pin the version of the SDK they are using. This was added to .NET Core as a way for users to roll-back to a previous version per project until a fix could be provided if there a problem occurred. While our guidance remains to use it only for special circumstances, users now use it for numerous reasons. 
```

Comment:
A nit, but this is a bit of mixing of metaphors. "pinning" and "roll-back" are different semantics and it's best to avoid evoking both of those terms in the same explanation. I think of global.json as "pinning". I don't think it has any "roll-back" semantics, at least with my definition of "roll-back", which I think of as being automatic and not manual.